src/euphorie/deployment/upgrade/alembic/versions/24_.py

```diff
@@ -0,0 +1,33 @@
+""" Try to add an image binary column and an image_filename column to the risk table
+
+Revision ID: 24
+Revises:
+Create Date: 2019-09-09 10:24:51.539339
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "24"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("risk", sa.Column("image_data", sa.LargeBinary(), nullable=True))
+    op.add_column(
+        "risk", sa.Column("image_data_scaled", sa.LargeBinary(), nullable=True)
+    )
+    op.add_column("risk", sa.Column("image_filename", sa.UnicodeText(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("risk", "image_data")
+    op.drop_column("risk", "image_filename")
```

Comment:
Just a doubt (not important): the downgrade should probable also drop column `image_data_scaled`?

Reply:
I think we will never used downgrades but anyway I pushed the requested change